testresources/all.json

```diff
@@ -698,6 +698,26 @@
       }
     }
   },
+  {
+    "type": "indicator",
```

Comment:
Not an incident

Reply:
Sorry, missed to add the changes in the commit 🤦‍♂️

all.json is now up to date

Reply:
Also checked the code with `go test -cover` and also added the assertions for all.json.
Coverage for this branch is now equal to master.